architecture/src/androidLibMain/kotlin/observable/AndroidObservable.kt

```diff
@@ -17,4 +17,17 @@
 
 package com.splendo.kaluga.architecture.observable
 
-actual class SimpleDisposable actual constructor(onDispose: DisposeHandler) : BaseSimpleDisposable(onDispose)
\ No newline at end of file
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Observer
+import androidx.lifecycle.asLiveData
+
+actual class SimpleDisposable actual constructor(onDispose: DisposeHandler) : BaseSimpleDisposable(onDispose)
+
+val <T>Uninitialized<T>.liveData:LiveData<T?>
+    get() = stateFlow.asLiveData()
+
+val <R:T, T>Initialized<R, T>.liveData:LiveData<R>
+    get() = stateFlow.asLiveData()
+
+val <R:T, T, OO:ObservableOptional<R>> BasicSubject<R, T, OO>.liveDataObserver: Observer<T>
```

Comment:
Two way binding is supported through `MutableStateFlow` by Android now, and observables / subjects have a `.stateFlow`

It would be good to also add something like this again though, maybe before merging to master. I created https://github.com/splendo/kaluga/issues/279

The problem with the old implementation is lack of testing, so it can't just be copied in so to say.